### PR TITLE
Ca 152 fixed text

### PR DESCRIPTION
**Pull request by JaycobDuffel**

### CODE REVIEW

**Review by DawsonDease — APPROVED** [review]

File: api/views/static.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from rest_framework import generics
+from api.models import Static
+from api.serializers import StaticSerializer
+
+
+class StaticTextListCreateView(generics.ListCreateAPIView):
+    queryset = Static.objects.all().order_by("id")
+    serializer_class = StaticSerializer
+
```
Comment (katiemcgoff) [inline api/views/static.py:9]: We also need the permissions class (see other endpoints) for both these views, to restrict to the admin user.

File: api/models/static.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        (THEME, "Theme"),
+        (QUOTE, "Quote"),
+        (GREETING, "Greeting"),
+        (TITLE, "Title"),
+    )
+    static_text_type = models.CharField(max_length=12, choices=TYPES_OF_STATIC_TEXT, default=THEME)
+    base_item = GenericRelation('BaseItem', related_query_name='static')
+
+    def __str__(self):
+        return self.name
```
Comment (katiemcgoff) [inline api/models/static.py:28]: there is no name field, so this should be something else - title probably.

**Review by katiemcgoff — CHANGES_REQUESTED** [review]
We also need api tests, so in the tests/api folder.

Otherwise very good, looks right and you checked all the boxes re. BaseItem.

**Comment by katiemcgoff** [comment]
Another test you should probably add to is the test_get_content() test in the UserFeed api test. It confirms that the UserFeed serializer changes work.

**Review by katiemcgoff — APPROVED** [review]

**Pull request merged**